src/components/Button/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import "./Button.scss";
+
+const Button = ({ buttonText, isEnterBtn }) => {
```

Comment:
`isEnterBtn` isn't very generic, is there a better way of referring to this different button? (secondary, alternate, large, rounded etc.)